README.md

```diff
@@ -1 +1,8 @@
-# behat-transformation-context
\ No newline at end of file
+# Behat Transformation Context
+
+[![Build Status](https://drone.fpfis.eu/api/badges/openeuropa/behat-transformation-context/status.svg)](https://drone.fpfis.eu/openeuropa/behat-transformation-context/)
+
+This package makes available a Context allowing to transform human readable labels to selectors or page paths.
+Such functionality improves the readability of the Behat tests.
+
+You can check included tests to see examples on how to use this context.
```

Comment:
I think we should add a "Usage" section where we explain how to use the extension in a project: adding it to composer, adding the context in the behat yml, an example configuration etc. We have been doing it in all other components so far